polyply/src/meta_molecule.py

```diff
@@ -217,8 +217,21 @@ def _block_graph_to_res_graph(block):
     @classmethod
     def from_monomer_seq_linear(cls, force_field, monomers, mol_name):
         """
-        Constructs a meta graph for a linear molecule
-        which is the default assumption from
+        Constructs a MetaMolecule from a list of monomers representing
+        a linear molecule.
+
+        Parameters
+        ----------
+        force_field: :class:`vermouth.forcefield.ForceField`
+            the force-field that must contain the block
+        monomers: list[:class:`polyply.meta_molecule.Monomer`]
+            a list of Monomer tuples
+        mol_name: str
+            name of the molecule
+
+        Returns
+        -------
+        :class:`polyply.src.meta_molecule.MetaMolecule`
```

Comment:
Is this how you access `MetaMolecule`? Would the following work?

```python
import polyply
polyply.src.meta_molecule.MetaMolecule
```

It looks like the `src` in the path should not be there.

Reply:
```suggestion
        :class:`polyply.MetaMolecule`
```

After a quick check, the `src` is correct, though it does not seem to be the way a user would import the object. See my code suggestion.